app/src/main/java/org/gdg/frisbee/android/app/App.java

```diff
@@ -245,4 +286,15 @@ public boolean isOrganizer() {
     public void checkOrganizer(GoogleApiClient apiClient, OrganizerChecker.OrganizerResponseHandler responseHandler) {
         mOrganizerChecker.checkOrganizer(apiClient, responseHandler);
     }
+
+    /**
+     * Return the current list of GDG event series occurring in the world.
+     * This may be empty but cannot be null.
+     *
+     * @return Array of current event series. 
+     */
+    @NonNull
+    public ArrayList<TaggedEventSeries> getTaggedEventSeriesList() {

```

Comment:
Remove Javadoc and rename method to `currentEventsOccurring()`
The problem with comments is that it's hard to make them correct or in sync with the code, for example, you wrote "The current list" and in the `@return` you say "Array of".